src/main/java/io/camunda/example/Authentication.java

```diff
@@ -1,9 +1,6 @@
 package io.camunda.example;
 
-import io.camunda.connector.api.annotation.Secret;
 import javax.validation.constraints.NotEmpty;
-import javax.validation.constraints.Pattern;
-
 import java.util.Objects;
 
 public class Authentication {
```

Comment:
What about using a record here?

Reply:
What about doing the same for the request class?

Reply:
That I am not sure about. It can get bigger, and might be inconvenient the lack of no-arg constructor, and structuring of attributes. But if you say these are not going to be issues then I can change that as well.

Reply:
I think it will be more consistent.

Reply:
Nice, looks even simpler with the records 😄